plotly/tools.py

```diff
@@ -2274,6 +2310,160 @@ def create_candlestick(open, high, low, close,
 
         return dict(data=data, layout=layout)
 
+    @staticmethod
+    def create_distplot(hist_data, group_labels,
+                        bin_size=1., curve_type='kde',
+                        show_hist=True, show_curve=True,
+                        show_rug=True, **kwargs):
+        """
+        BETA function that creates a distplot similar to seaborn.distplot
+
+        The distplot can be composed of all or any combination of the following
+        3 components: (1) histogram, (2) curve: (a) kernal density estimation
+        or (b)normal curve, and (3)rug plot. Additionally, multiple distplots
+        (from multiple datasets) can be created in the same plot.
+
+        :param (list) hist_data: list of histogram data, should be a list of
+            lists if multiple data sets are plotted on the same plot
+        :param (list) group_labels: list of strings of the name of each data
+            set
+        :param (float) bin_size: size of histogram bin, default = 1.
+        :param (string) curve_type: 'kde' or 'normal', default = 'kde'
+        :param (bool) show_hist: True/False defines if histogram is added to
+            distplot, default = True
+        :param (bool) show_curve: True/False defines if curve is added to
+            distplot, default = True
+        :param (bool) show_rug: True/False defines if rug plot is added to
+            distplot, default = True
+        :param (list) colors: list of strings of color values,
+            default = first 10 Plot.ly colors
+        :param (list) rug_text: list of strings of hovertext for rug_plot,
+            default=None
+
+        :rtype (dict): returns a representation of a distplot figure.
+
+        Example 1: Simple distplot of 1 data set
+        ```
+        import plotly.plotly as py
+        from plotly.tools import FigureFactory as FF
+
+        hist_data = [[1.1, 1.1, 2.5, 3.0, 3.5,
+                      3.5, 4.1, 4.4, 4.5, 4.5,
+                      5.0, 5.0, 5.2, 5.5, 5.5,
+                      5.5, 5.5, 5.5, 6.1, 7.0]]
+
+        group_labels = ['distplot example']
+
+        fig = FF.create_distplot(hist_data, group_labels)
+
+        url = py.plot(fig, filename='Simple distplot', validate=False)
+        ```
```

Comment:
I think that the annotation is probably redundant, since it's already color coded in the legend